draft-kwbdgrr-tsvwg-net-collab-rqmts.md

```diff
@@ -70,14 +70,16 @@ This document lists some use cases that demonstrate the need for a mechanism to
 
 Wireless networks inherently experience large variations in link capacity due to several factors.
 These include the change in wireless channel conditions, interference between proximate cells and channels or because of the end user movement.
-These variations in link capacity can be in the order of a millisecond or less.
-Media frames may experience high sojourn times in the wireless network or the flow may experience lower throughput, neither of which is desirable.
+These variations in link capacity can be in the order of a millisecond or less which is far lower than RTT.
+End-to-end congestion control algorithms are far from optimal when the link capacity is highly variable in sub-RTT timeframes and the application demands both low latency and high bandwidth. 
```

Comment:
can we have a reference to cite here?

Reply:
Probably many references, but two that are good:
1. 5G: Lumos5G: Mapping and Predicting Commercial mmWave 5G Throughput, Arvind Narayanan et al., ACM https://dl.acm.org/doi/10.1145/3419394.3423629
2. LTE: Stochastic Forecasts Achieve High Throughput and Low Delay over Cellular Links, Winstein et al., USENIX NSDI, https://www.usenix.org/system/files/conference/nsdi13/nsdi13-final113.pdf

In the draft, we could add (1) which is more recent.